build/config/mac/mac_sdk.gni

```diff
@@ -10,7 +10,7 @@ declare_args() {
 
   # The MACOSX_DEPLOYMENT_TARGET variable used when compiling.
   # Must be of the form x.x.x for Info.plist files.
-  mac_deployment_target = "10.11.0"
+  mac_deployment_target = "10.12.0"
```

Comment:
Can you change this to 10.13?  See https://github.com/flutter/flutter/issues/107492#issuecomment-1185054687

Reply:
done

Reply:
Should `mac_sdk_min` also be bumped to 10.13?

Reply:
I guess it should if everything happily compiles...

Reply:
Updated `mac_sdk_min` to 10.13 and confirmed that the engine builds and runs